### PR TITLE
p2o fix patch added for da-ds

### PR DESCRIPTION
**Pull request by lfpratik**

### CODE REVIEW

File: gitops.go [hunk 1]
```diff
@@ -360,35 +360,36 @@ func (g *gitOps) loc(value string, force bool) int64 {
...
 		defer func() {
 			fmt.Printf("pls %s ---> %+v\n", value, stats)
 		}()
 	}
-	if strings.Contains(value, "SUM:") || force {
+	if strings.Contains(value, "SUM:") || strings.Contains(value, "Language") {
 		lanSmryLst := strings.Split(value, "\n")
 		nLanSmryLst := len(lanSmryLst)
-		for i := nLanSmryLst - 1; i >= 0; i-- {
+		var hasLanguage bool = false
```
Comment (lukaszgryglicki) [inline gitops.go:373]: Please replace `var hasLanguage bool = false` with `hasLanguage := false`

File: gitops.go [hunk 2]
```diff
@@ -360,35 +360,36 @@ func (g *gitOps) loc(value string, force bool) int64 {
...
+		for i := 0; i <= nLanSmryLst-1; i++ {
```
Comment (lukaszgryglicki) [inline gitops.go:374]: Loop order was reversed, is it intended?

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm, but please note my 2 comments.
Let me know if I should build a new `test` SDS for this.

File: gitops.go (hunk 2 above)
Comment (lfpratik) [reply gitops.go:374]: Yes

File: gitops.go (hunk 1 above)
Comment (lfpratik) [reply gitops.go:373]: I will do it.

Comment (lukaszgryglicki) [reply gitops.go:373]: Please merge when done.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**